start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js

```diff
@@ -0,0 +1,725 @@
+function NewEvaluationPage(options){
+    
+    const elements = [
+        {level:'Trainee'},{level:'Junior'},{level:'Middle'}, {level:'Senior'}
+    ];
+
+    const selector = [
+        {
+            value:'1',
+            language: 'Evaluation',
+        },
+        {
+            value:'2',
+            language: 'javascript',
+        },
+        {
+            value:'3',
+            language: 'Java',
+        },
+        {
+            value:'4',
+            language: 'Haskell',
+        }
+    ];
+
+    const description = [
+        {
+            selectDescription: 'Evaluation'
+        },
+        {
+            selectDescription: 'Exception handling'
+        },
+        {
+            selectDescription: 'Version Control'
+        },
+        {
+            selectDescription: 'Access modifiers'
+        },
+        {
+            selectDescription: 'Design Pattern'
+        },
+        {
+            selectDescription: 'Design Pattern'
+        },
+        {
+            selectDescription: 'Polymorphism'
+        },
+        {
+            selectDescription: 'RegEx'
+        }
+
+    ];
+
+
+    const newOptions = options || {};
+    return `
+    ${NAV()}
+    ${NewEvaluationTable({
+        el: elements,
+        sel: selector,
+        des: description
+    })}
+    ${FooterNew()}
+
+
+    `;
+}
+
+const TechnicalLevel = function(options={}){
+    const technical = [];
+    options.el.forEach(function(n){
+        technical.push(`<td class='technical-level-td'>${n.level}</td>`);
+    });
+    return technical.join('');
+    
+}
+
+const SelectLevel = function(options={}){
+    const selectArr = [];
+    options.sel.forEach(function(n){
+        selectArr.push(`
```

Comment:
Do you really need the extra newlines here? The browser would still know how to interpret / display this, even if all of your HTML is compressed onto one line (well, maybe not all... preformatted elements would still require proper spacing to display as intended, but anywho, for the most part, it'll look good either way).

Reply:
Thanks for pointing this out. I managed to refactor that function.

Reply:
I refactored this. Thanks for pointing this out.